### ISSUE TITLE
Div does not respect height and width opts

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/holoviews#4808]
Holoviews: 1.14.1

I'm having getting the combined functionality of a candle stick plot and two histograms working. I can lay it out using a Panel gridspec with HoloViews panes as I want. But when I update the objects of the HoloViews panes the axis ranges do not update accordingly. 

So now I am trying if laying things out in a HoloViews Layout will make things work. Here it seems the axis updates but now the plots do not layout as I want.

In the below example I would expect the `Div` in blue to have a `width` and `height` of 100. But it has not.

```python
import holoviews as hv
ul = hv.Curve({}).opts(height=100, responsive=True)
ur = hv.Div("<div style='height:100%;width:100%;background:blue;'></div>").opts(height=100, width=100)
ll = hv.Curve({}).opts(responsive=True, min_height=300)
lr = hv.Curve({}).opts(width=100, responsive=True)
(ul + ur + ll + lr).cols(2)
```
![image](https://user-images.githubusercontent.com/42288570/106352295-2c485180-62e2-11eb-8445-b958106361e0.png)

**Comment by MarcSkovMadsen** [comment]
## Additional Context

If I replace the `Div` with a `Curve` the Layout works

![image](https://user-images.githubusercontent.com/42288570/106352348-a11b8b80-62e2-11eb-8ad4-197b5e5c5a62.png)

```python
import holoviews as hv
ul = hv.Curve({}).opts(height=100, responsive=True)
ur = hv.Curve({}).opts(height=100, width=100)
ll = hv.Curve({}).opts(responsive=True, min_height=300)
lr = hv.Curve({}).opts(width=100, responsive=True)
(ul + ur + ll + lr).cols(2)
```

**Comment by jlstevens** [comment]
I agree something is off as the div is only declaring the size in percent and the `.opts` specify the absolute height/width. Tagging as a bug.
